frontend/src/metabase/dashboard/components/Dashboard/Dashboard.jsx

```diff
@@ -73,6 +69,11 @@ class DashboardInner extends Component {
 
     if (
       !_.isEqual(prevProps.parameterValues, this.props.parameterValues) ||
+      (!this.props.isEditing &&
```

Comment:
I have updated the conditional so that we now check for `isEdited`. Do you think it's enough?

Reply:
What can be changing dashcards outside of edit mode?

Reply:
Another comment along the way: IMO, this condition block became too complex. We should extract parts of the expression into constants with human-readable names. I'm thinking of something like:

```ts
const didParameterValuesChange = !_.isEqual(prevProps.parameterValues, this.props.parameterValues);
const didDashCardsChange = !_.isEqual(
  prevProps.dashboard?.dashcards,
  this.props.dashboard?.dashcards,
))
const didLoadDashboard = !prevProps.dashboard && this.props.dashboard;

if (
  didLoadDashboard ||
  didParameterValuesChange ||
  (!isEditing && didDashCardsChange)
) { ... }
```